### DISCUSSION TITLE
Contributing: guidelines & prerequisite?

### DISCUSSION THREAD

**Discussion by yscialom** [AdguardTeam/AdGuardHome#4827] [Q&A] [answered]
Hello,

What are the prerequisite for contributing to AdGuardHome?

I recently raised an issue and proposed a change to fix it. After 5 weeks I still had no feedback whatsoever.

Maybe I missed a guideline, or my issue+PR is wrong in a way, or maybe it tackles a subject the core team has no intention to touch right now?

If you're curious, I refer to issue/4714 and PR #4728, but my question is more general as I may be contributing more and more to the product.

BW,
Yankel.

**Comment by ainar-g** [comment, marked as answer]
Good day.  Apologies for the slow response times.  Our Go team has unfortunately been extremely busy working on both on AdGuard DNS as well as other internal projects, as you can probably tell by the number of PRs waiting to be reviewed.  I've added your PR to the next milestone, and we'll try to review and publish it when we finally have the time.

**Edit:**  A contribution guideline is also definitely in our TODO with regards to documentation and project management generally.

**Reply by yscialom** [reply]
Hello ainar, and thank you for the prompt answer.

Now that I understand you are busy right now, the waiting time will be way less of a hassle for me. Good luck with all the pressure, and keep the good work.
